doc/users/resources/index.rst

```diff
@@ -85,3 +84,13 @@ Tutorials
 * `Beyond the Basics: Data Visualization in Python
   <https://github.com/stefmolin/python-data-viz-workshop>`_
   by Stefanie Molin
+
+=========
+Galleries
+=========
+
+
+[Past winners for JDH plotting contest](https://jhepc.github.io/gallery.html)
```

Comment:
Can you bullet point this one as well? Also, this file uses RST syntax, not Markdown. You can learn about RST here: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html

Reply:
Thanks for pointing that out. I has